github/repos.go

```diff
@@ -529,6 +531,12 @@ type RequiredStatusChecks struct {
 	Contexts *[]string `json:"contexts,omitempty"`
 }
 
+// RequiredPullRequestReviews represents the protection configuration for pull requests.
+type RequiredPullRequestReviews struct {
+	// Enforce pull request reviews for repository administrators.
+	IncludeAdmins *bool `json:"include_admins,omitempty"`
```

Comment:
> I like that it exposes the requirements of the API in the type itself. I've made it so.

Great!

> If you think it's a bad idea, though, I can lop that commit off.

Not at all, on the contrary, I think that's absolutely the way to go. Thank you for doing that.

Yes, the GitHub API is still in beta so I think it's fine to break it slightly, since it's an improvement.

/cc @varadarajana FYI, see above. You'll need to make a small change to your PR #509 as a result.